src/spawn_sync.cc

```diff
@@ -650,12 +650,16 @@ Local<Object> SyncProcessRunner::BuildResultObject() {
                    Integer::New(env()->isolate(), GetError()));
   }
 
-  if (exit_status_ >= 0)
-    js_result->Set(env()->status_string(),
-        Number::New(env()->isolate(), static_cast<double>(exit_status_)));
-  else
+  if (exit_status_ >= 0) {
+    if (term_signal_ > 0)
+      js_result->Set(env()->status_string(), Null(env()->isolate()));
+    else
```

Comment:
I know it didn't have them originally, but can we please add braces here (and around the `if` block for completeness), since the block spans more than one line?